.buildkite/commands/run-unit-tests.sh

```diff
@@ -11,5 +11,26 @@ gem install bundler
 echo "--- :rubygems: Setting up Gems"
 install_gems
 
-echo "--- 🧪 Testing"
+echo "--- 🔬 Testing"
+set +e
 bundle exec fastlane test_without_building name:WordPressUnitTests try_count:3
+TESTS_EXIT_STATUS=$?
+set -e
+
+if [[ "$TESTS_EXIT_STATUS" -ne 0 ]]; then
```

Comment:
Nitpick: Below this same check is without quotes and it works.

```suggestion
if [[ $TESTS_EXIT_STATUS -ne 0 ]]; then
```

I'm not an expert on Bash scripting, but I think the main value in quoting variables is to avoid issues when they are strings and have spaces in them. This is a return code, a number, so we should be fine.

Reply:
That's indeed probably fine given that `TESTS_EXIT_STATUS` is always an integer, not a string with potential spaces and all. That being said, I like to always use quotes, first just to take the good habit of doing so in all cases, but also to make [`shellcheck`](https://github.com/koalaman/shellcheck#user-content-installing) (the `swiftlint` or `rubocop` of shell scripts) happy(†)

(†) Though (1) I'm not sure if `shellcheck` might not be able to detect that this variable is always an integer and be smart enough to no require quotes in that case 🤔  and (2) I think `[[ … ]]` is also smarter w.r.t. those cases as opposed to `[ … ]`. But as long as `shellcheck` does not complain, I'm confident that the current solution here is safe

Reply:
Mmmh actually [according to SO, it seems that quoting is unnecessary when using `[[`](https://stackoverflow.com/a/3427931) (as opposed to `[`)… regardless of the variable tested being a number or a string 🤯 TIL

Anyway, both work for me, so I don't mind keeping the quotes just to make this consistent with how it was done in the `run_ui_tests.sh` counterpart 🤷‍♂️

Reply:
Always learning something about Bash. =P
Moved on with `[[ ... ]]` without quoting: [bbab751](https://github.com/wordpress-mobile/WordPress-iOS/pull/17999/commits/bbab75175d3c595d58526cad4a20a9b5b03676f1)